src/provenance/LocalStorageProvenanceGraphManager.ts

```diff
@@ -12,15 +12,27 @@ import {UserSession} from '../app/UserSession';
 import {Permission} from '../security/Permission';
 import {MemoryGraph} from '../graph/MemoryGraph';
 import {ResolveNow} from '../base/promise';
+import {ICommonProvenanceGraphManagerOptions} from '.';
+
+export interface ILocalStorageProvenanceGraphManagerOptions extends ICommonProvenanceGraphManagerOptions {
+  storage?: Storage;
+  prefix?: string;
```

Comment:
Please add some documentation to these properties.

Reply:
These options where taken directly from the (previously undocumented) options here: https://github.com/phovea/phovea_core/blob/develop/src/provenance/LocalStorageProvenanceGraphManager.ts#L17-L21

I honestly do not know the exact workings of these three settings are, maybe you (or any other person?) does know what they are doing beyond the obvious?

Reply:
> beyond the obvious

😄 

I will try to document it.